src/lib/cli.ts

```diff
@@ -150,90 +61,13 @@ export const execute = async (args: string | Array<string> | Object): Promise<nu
         notifyNewVersion(update);
     }
 
-    if (currentOptions.init) {
-        await initSonarrc();
-
-        return 0;
-    }
-
-    if (currentOptions.newRule) {
-        await newRule();
-
-        return 0;
-    }
-
-    if (currentOptions.removeRule) {
-        await removeRule();
-
-        return 0;
-    }
-
-    if (currentOptions.help || !targets.length) {
-        logger.log(options.generateHelp());
-
-        return 0;
-    }
-
-    let config: IConfig;
-    const configPathFromInput: string = currentOptions.config;
-
-    if (configPathFromInput) {
-        config = Config.load(configPathFromInput);
-    } else {
-        config = await loadOrCreateIfNotExits();
-    }
+    let handled = false;
 
-    if (!config) {
-        logger.log(`Unable to find a valid configuration file. Please add a .sonarrc file by running 'sonar --init'. `);
+    while (cliActions.length > 0 && !handled) {
```

Comment:
Some of the actions can throw exceptions. It think it would be nice to have a `try/catch` wrapping the `while`

Reply:
Change is pushed. Wrapping individual actions instead of the `while` though.